app/routes/docs.vtp.mdx

```diff
@@ -27,19 +39,22 @@ This migration plan addresses only the transport protocol, not the XML data seri
 
 Organizations including the new GCN, Rubin, SCiMMA, have moved away from VTP in favor of more modern, general-purpose data distribution protocols like [Kafka](client). Kafka is widely used throughout industry, is well supported, and is growing in its use in astronomy. As GCN services move from GCN Classic to the new GCN, we are planning to retire the existing GCN Classic VOEvent system as it is currently implemented in several cloud-based servers that do not meet NASA standards. The GCN team is gradually migrating to or replacing GCN Classic services in the new GCN, and is not planning any new development in GCN Classic.
 
-We intend for this transition to be planned, announced, and implemented with ample warning for the user community. We propose 3 alternatives for how to complete this transition, and are soliciting [user feedback](vtp#feedback-survey). We know that many pipelines, robotic telescopes, and users depend on this service and we aim to be minimally disruptive to their operations.
+The result of the November 2024 user survey of those utilizing the GCN VOEvent Brokers to receive VOEvent-format Notices with the VOEvent Transport Protocol is overwhelmingly in favor of retiring the brokers and users migrating to receive VOEvent Notices over Kafka. This service is already available for all GCN Classic Notice types with self-subscription via the [Start Streaming Notices Guide](/quickstart).
+
+## GCN VTP Migration
+
+[As announced in February 2025](/news#planned-retirement-of-gcn-classic-voevent-brokers), the [GCN Classic VOEvent brokers](https://gcn.gsfc.nasa.gov/voevent.html) will be retired shortly after the conclusion of the O4 gravitational wave network observing run, [currently scheduled for October 7, 2025](https://observing.docs.ligo.org/plan/). We recommend the following actions for users currently utilizing VOEvent via GCN Classic. - Sign up to receive VOEvent format Notices via [Kafka or email](/docs/notices/consuming) - [Let the GCN Team know](/contact) when you're ready to unsubscribe from receiving them via the GCN Classic brokers. All GCN Classic VOEvent subscribers will be unsubscribed after the end of O4 and the brokers will be taken offline.
```

Comment:
Looks like you meant there to be a list here, but it's smooshed into a paragraph.

Reply:
Whoops, you are correct. I've fixed this.